CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+### v1.1.0 / 2024-10-22
+
+[Commit list](https://github.com/intel/ledmon/compare/v1.0.0...v1.1.0)
+
+Enhancements
+* tests: improve subprocess handling in run_ledctl_cmd()
+* Add struct for mapping ibpi statuses
+* ci: add hardening-check action
+* Improve github actions workflows
+* Update licensing information and documentation
+* Add support for gcc 14
+* Enhance configure.ac for better gcc flags validation
+* Add --default-controller command to ledctl
```

Comment:
This is real feature. For other you can just:
- various enhancements in tests, licensing and deployment.